Integrations/MISP_V2/MISP_V2.py

```diff
@@ -128,6 +125,71 @@ def warn(*args):
 ''' HELPER FUNCTIONS '''
 
 
+def filter_obj(obj: dict, is_att: bool = True) -> dict:
+    """ Filter Event, Attribute
+
+    Args:
+        obj(dict): dictionary MISP event or attribute.
+        is_att: True if dict represent Attribute otherwise Event (False)
+
+    Returns:
+        dict: Filtered dict by configuration.
+    """
+    metadata_state = demisto.params().get('metadata')
+    related_events_state = demisto.params().get('related_events')
```

Comment:
are we trying to save BC here? because the way this is implemented this will surly not.